lib/matplotlib/backends/backend_template.py

```diff
@@ -212,19 +207,18 @@ def new_figure_manager_given_figure(num, figure):
 class FigureCanvasTemplate(FigureCanvasBase):
     """
     The canvas the figure renders into.  Calls the draw and print fig
-    methods, creates the renderers, etc...
+    methods, creates the renderers, etc.
 
-    Note GUI templates will want to connect events for button presses,
+    Note: GUI templates will want to connect events for button presses,
     mouse movements and key presses to functions that call the base
     class methods button_press_event, button_release_event,
-    motion_notify_event, key_press_event, and key_release_event.  See,
-    e.g., backend_gtk.py, backend_wx.py and backend_tkagg.py
+    motion_notify_event, key_press_event, and key_release_event.  See the
+    implementations of the interactive backends for examples.
```

Comment:
Can we point to a more concrete location?

Reply:
Literally any of the interactive backends will do; I don't think there's much of a point of picking one versus the other.